res/values/strings_tl.arb

```diff
@@ -0,0 +1,686 @@
+  "enter_backup_password" : "Ilagay ang backup na password dito",
+  "select_backup_file" : "Piliin ang backup na file",
+  "import" : "Umangkat",
+  "please_select_backup_file" : "Mangyaring piliin ang backup na file at ilagay ang backup na password.",
+
+  "fixed_rate" : "Fixed rate",
+  "fixed_rate_alert" : "Makakapagpasok ka ng halaga ng pagtanggap kapag nasuri ang fixed rate mode. Gusto mo bang lumipat sa fixed rate mode?",
+
+  "xlm_extra_info" : "Mangyaring huwag kalimutang tukuyin ang memo ID habang ipinapadala ang transaksyon sa XLM para sa palitan",
+  "xrp_extra_info" : "Mangyaring huwag kalimutang tukuyin ang patutunguhang tag habang ipinapadala ang XRP na transakyson para sa palitan",
+
+  "exchange_incorrect_current_wallet_for_xmr" : "Kung gusto mong palitan ang XMR mula sa iyong balanse ng Cake Wallet Monero, mangyaring lumipat muna sa iyong Monera wallet.",
+  "confirmed" : "Nakumpirma",
+  "unconfirmed" : "Hndi kumpirmado",
+  "displayable" : "Maipapakita",
+
+  "submit_request" : "Magsumite ng kahilingan",
+
+  "buy_alert_content" : "Sa kasalukuyan ay sinusuportahan lamang namin ang pagbili ng Bitcoin at Litecoin. Para bumili ng Bitcoin or Litecoin, mangyaring lumikha o lumipat sa iyong Bitcoin o Litecoin wallet.",
+  "sell_alert_content": "Sa kasalukuyan ay sinusuprtahan lamang namin ang pagbebenta ng Bitcoin. Para magbenta ng Bitcoin, mangyaring lumikha o lumipat sa iyong Bitcoin wallet.",
+
+  "outdated_electrum_wallet_description" : "Ang mga bagong Bitcoin wallet na ginagawa sa Cake ay mayroon na ngayong 24 na salita na seed. Ipinag-uutos na lumikha ka ng bagong bitcoin wallet at ilipat ang lahat ng iyong pondo sa bagong 24-salitang wallet, at ihinto ang paggamit ng mga wallet na may 12-salitang seed.Mangyaring gawin ito kaagad upang ma-secure ang iyong mga pondo",
+  "understand" : "Naiitindihan ko",
+
+  "apk_update" : "Pag-update APK",
+
+  "buy_bitcoin" : "Bumili ng Bitcoin",
+  "buy_with" : "Bumili ng",
+  "moonpay_alert_text" : "Ang halaga ay dapat na higit pa o katumbas ng ${minAmount} ${fiatCurrency}",
+
+  "outdated_electrum_wallet_receive_warning": "Kung ang wallet na ito ay may 12-word seed na ginawa sa Cake, huwag magdeposito ng Bitcoin sa wallet na ito. Anumang BTC na inilipat sa wallet na ito ay maaaring mawala. Lumikha ng bagong 24 na salita na wallet (i-tap ang menu sa kanang taas, piliin ang Mga Wallets, piliin ang Gumawa ng Bagong Wallet, pagkatapos ay piliin ang Bitcoin) at agad na ilipat ang iyong BTC doon. Bagong (24 na salita) BTC wallet mula sa Cake ay secured",
+  "do_not_show_me": "Huwag mo nang ipakita sa akin ang mensahing ito",
+
+  "unspent_coins_title" : "Hindi nagamit na mga barya",
+  "unspent_coins_details_title" : "Hindi nagastos na mga detalye ng barya",
+  "freeze" : "Freeze",
+  "frozen" : "Frozen",
+  "coin_control" : "Kontrol ng barya (opsyonal)",
+
+  "address_detected" : "Nakita ang address",
+  "address_from_domain" : "Ang address na ito ay mula sa ${domain} mga Unstoppabe Domains",
```

Comment:
Little typo, the letter "l" is missing, it should be *Unstoppable Domains*

Reply:
🙈